functions.opengraph.php

```diff
@@ -261,7 +274,8 @@ function jetpack_og_tags() {
 		}
 	}
 	$og_output .= "\n<!-- End Jetpack Open Graph Tags -->\n";
-	echo $og_output;
+	// This is trusted output or added by a filter.
+	echo $og_output; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
I don't like ignoring this here, but I don't want to run `wp_kses` either since it is potentially expensive for front-end output.

The alternative would be a pretty decent reworking of how we add things to $og_output to foreach echo out the meta tags based on an array or something, which would break BC with at least one filter.

I'm sure I'm missing something though.